docs/_docs/focus/validate.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+parent: FOCUS
+title: Validate FOCUS data
+nav_order: 3
+description: 'Validate FOCUS data with existing Cost Management datasets.'
+permalink: /focus/validate
+---
+
+<span class="fs-9 d-block mb-4">Validate FOCUS data</span>
+Compare and validate FOCUS cost and usage data with existing Cost Management actual and amortized datasets.
+{: .fs-6 .fw-300 }
+
+<details open markdown="1">
+   <summary class="fs-2 text-uppercase">On this page</summary>
+
+- [📋 Comparing FOCUS with actual cost](#-comparing-focus-with-actual-cost)
+- [📋 Comparing FOCUS with amortized cost](#-comparing-focus-with-amortized-cost)
+- [🙋‍♀️ Feedback about FOCUS columns](#️-feedback-about-focus-columns)
+- [🧐 See also](#-see-also)
+- [🧰 Related tools](#-related-tools)
+
+</details>
+
+---
+
+This document helps you validate FOCUS data by comparing it to actual and amortized cost data from Cost Management. The goal of this article is to help you familiarize yourself with and transition to FOCUS data with confidence. To learn more about FOCUS, refer to the [FOCUS overview](./README.md).
+
+The comparisons in this document are focused on costs, prices, and quantities to facilitate validating numbers across datasets. For details about text and dates, refer to [Updating reports to use FOCUS columns](./mapping.md).
+
+Note that FOCUS includes both actual and amortized data in a single dataset so there will be more rows in FOCUS than you will find in either the actual or amortized cost datasets, but there are fewer records than actual and amortized datasets together since many records are duplicated across actual and amortized datasets.
+
+<br>
+
+## 📋 Comparing FOCUS with actual cost
+
+The following mapping can be used to compare FOCUS with an actual cost data in Cost analysis or other cost API or export.
+
+| Cost Management column                                                                                      | FOCUS column                                                                    | Notes                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
To make it a bit more obvious, can we rename Cost Management column as Cost Management column (Actual cost)?

Reply:
Love the idea!